README.md

```diff
@@ -67,17 +78,18 @@ resource "aws_ami_from_instance" "example" {
 | `security_groups`            | []             | List of Security Group IDs allowed to connect to creating instance        | Yes |
 | `subnets`                    | []             | List of VPC Subnet IDs creating instance launched in                      | Yes |
 | `zone_id`                    | ``             | ID of the domain zone to use - is a result of tf_domain output            | Yes |
+| `associate_public_ip_address`| `true`         | Define if the `public_ip` will be created for the instance                | No  |
 
 ## Outputs
 
-| Name                | Decription              |
-|:-------------------:|:-----------------------:|
-| `id`                | Disambiguated ID        |
-| `fqdn`              | Normalized name         |
-| `public_ip`         | Normalized namespace    |
-| `ssh_key_pair`      | Name of used AWS SSH key|
+| Name                | Decription                                                        |
+|:-------------------:|:-----------------------------------------------------------------:|
+| `id`                | Disambiguated ID                                                  |
+| `fqdn`              | Normalized name                                                   |
```

Comment:
@comeanother So you mean I need to decrypt `fqdn` as 'Fully Qualified Domain Name'?